ui/yarn.lock

```diff
@@ -5825,7 +5825,7 @@ mkdirp@^0.5.1:
   dependencies:
     minimist "^1.2.6"
 
-moment-timezone@^0.5.34, moment-timezone@^0.5.43:
```

Comment:
this is still listed as a dep in [`argo-ui`](https://github.com/argoproj/argo-ui/blob/5ff344ac9692c14dd108468bd3c020c3c75181cb/package.json#L32) although it's actually [unused there](https://github.com/search?q=repo%3Aargoproj%2Fargo-ui%20moment-timezone&type=code), hence why it doesn't show up in the bundle anymore.

I'll have some separate PRs to clean-up more of `argo-ui`, it's uh, got a few things up with its `package.json` (like deps + peerDeps overlapping and also having different versions 😕 )

Reply:
Made one upstream fix here: https://github.com/argoproj/argo-ui/pull/511. I'll have a few more follow-ups to that as they may merge conflict with each other

Reply:
Welp that still hasn't been reviewed or merged, but added https://github.com/argoproj/argo-ui/pull/534 as well (which removes the unused `moment-timezone` plus a bunch of other unused deps). Seems to actually not conflict, though I wrapped together a few of my local changes into that one.